cmd/installer/cli/install.go

```diff
@@ -1106,7 +1109,9 @@ func verifyLicense(license *kotsv1beta1.License) (*kotsv1beta1.License, error) {
 		return nil, err
 	}
 
-	if isV3Enabled() {
+	// Skip signature verification in dryrun mode since test licenses don't have valid signatures
+	// TODO: assert this is called in dryrun mode
```

Comment:
hmm we should fix dryrun tests to use valid licenses... let's create a story at least to track this?